_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+
+
+...El caso del archivo de Gabriel García Márquez también nos abre a la reflexión sobre lo que significa un archivo per se debido a la polémica que se generó cuando la familia decidió vendérselo al Harry Ransom Center.
+
+Entonces vamos más allá del archivo. Más allá de una simple recolección de papeles depositados en 24 cajas. El archivo como bien lo describían los autores Carlos Acuña y Javier Villa Flores en el texto “Los archivos y la construcción de la verdad histórica en América Latina” son:	
+		
+ *“Lejos de ser depósitos imparciales, los archivos operan a partir de relaciones de autoridad y fuerza que determinan lo que es “archivable” y lo que es “accesible”. Su propia creación, contenido y funcionamiento son el resultado de acciones deliberadas de inclusión y exclusión tendientes a “organizar”, “ordenar” y, por lo mismo, construir” el pasado.”*...
+ 
+
+...El archivo significa un pasado, un presente y un futuro. Y en este sentido es comprensible que el traslado, por ejemplo, de estos archivos a otro país haya generado tanta polémica...
+
+
+...Cuando el escritor murió en el 2014 y se dió a conocer la noticia del destino de sus archivos, muchas fueron las reacciones a favor y en contra de la decisión. La idea de que el escritor les pertenece a los colombianos se diluyó al vender su memorabilia a un centro en los Estados Unidos. Yo diría que con no sólo con esta decisión García Márquez dejó de ser de los colombianos, sino que una vez que se digitalizó el archivo, se perdió en el mundo global del internet. Ahora el escritor está en la nube para todo todo el mundo. Todos los que tiene acceso a internet pueden gozar de la presencia de él en su espacio.
+
+Como consecuencia, el archivo se convirtió en algo etéreo y concreto a la vez...
+
+
+...El archivo deja de tener propiedad aunque legalmente la tenga. La autoridad del archivo fue trasladada a otro sujeto y a otro espacio o a muchos sujetos y a muchos espacios...
+
+
+...Cuando se discutió este tema sobre el destino de los archivos del premio Nobel muchas preguntas salieron al debate público:  hizo el gobierno colombiano lo necesario para quedarse con los archivos?  Podría invertir el gobierno o alguien en Colombia 2,2 millones de dólares para la adquisición de este material? Por qué el gobierno colombiano permitió que se llevaran esos archivos?  Si estuviese en Colombia se preservarían de la misma forma? Se digitalizarían?
+
+Ninguna de estas preguntas tendrán respuestas, pero sí nos permiten la reflexión sobre lo que puede significar un archivo, lo que significa digitalizarlo, el significado de la accesibilidad del mismo y lo que además representa migrar los documentos a otro país , es decir, “desarraigarlo” de su entorno...
```

Comment:
I'd like you to reflect a little more deeply on this question. Do you think the digital project would look different if it had been done by a colombian archive? How?